packages/kit/test/apps/basics/src/routes/__layout.svelte

```diff
@@ -9,6 +11,13 @@
 			}
 		};
 	}
+
+	if (browser) {
+		let h3 = document.createElement('h3');
+		let text = document.createTextNode('You cannot reset me');
```

Comment:
I don't understand this message "You cannot reset me". It looks like the test is checking that the h3 is not appended because the layout is reset. In that case, would a better message be "I should not be rendered" or something? Or maybe I'm misunderstanding how the test is actually working?

Reply:
You understood it completely correct. I have changed the message.